manuscript/developing/02_development_server.md

```diff
@@ -2,46 +2,54 @@
 
 If you go with WDS, there are a couple of relevant fields that you should be aware of:
 
 - `devServer.historyApiFallback` should be set if you rely on HTML5 History API based routing.
 - `devServer.contentBase` - Assuming you don't generate _index.html_ dynamically and prefer to maintain it yourself in a specific directory, you need to point WDS to it. `contentBase` accepts either a path (e.g., `"build"`) or an array of paths (e.g., `["build", "images"]`). The value defaults to the project root.
-- `devServer.proxy` - If you are using multiple servers, you have to proxy WDS to them. The proxy setting accepts an object of proxy mappings (e.g., `{ "/api": "http://localhost:3000/api" }`) that resolve matching queries to another server. Proxy settings are disabled by default.
+- `devServer.proxy` - If you are using multiple servers, you have to proxy WDS to them. The proxy setting accepts an object of proxy mappings (e.g., `{ "/api": "http://localhost:3000/api" }`) that resolve matching queries to another server. Proxying is disabled by default.
 - `devServer.headers` - Attach custom headers to your requests here.
 
 T> [The official documentation](https://webpack.js.org/configuration/dev-server/) covers more options.
 
 T> To integrate with another server, it's possible to emit files from WDS to the file system by setting `devServer.writeToDisk` property to `true`.
 
-W> You should use WDS strictly for development. If you want to host your application, consider other standard solutions, such as Apache or Nginx.
+W> You should use WDS strictly for development. If you want to host your application, consider other solutions, such as Apache or Nginx.
 
-W> WDS depends implicitly on **webpack-cli** in command line usage. Make sure you have both installed in this case.
+W> WDS depends implicitly on **webpack-cli** in command line usage. Make sure you have both installed.
 
 ## **webpack-plugin-serve**
 
-[webpack-plugin-serve](https://www.npmjs.com/package/webpack-plugin-serve) (WPS) is a third-party plugin that wraps the logic required to update the browser into a webpack plugin. Underneath it relies on webpack's watch mode, and it builds on top of that while implementing **Hot Module Replacement** (HMR) and other features seen in the official solution provided for webpack.
+[webpack-plugin-serve](https://www.npmjs.com/package/webpack-plugin-serve) (WPS) is a third-party plugin that wraps the logic required to update the browser into a webpack plugin. Underneath it relies on webpack's watch mode, and it builds on top of that while implementing **Hot Module Replacement** (HMR) and other features seen in the **official solution provided for webpack (AS: which one do you mean here? WDS or --watch?)**.
+
+WPS also supports webpack's multi-compiler mode (i.e., when you give an array of configurations to it) and a status overlay.
+
+**AS: if it's based on --watch, does it mean it works on disk files and slower than WDS?**
```

Comment:
I haven't noticed a significant difference in my usage.

That said, there's support for using a ramdisk: https://www.npmjs.com/package/webpack-plugin-serve#ramdisk and likely we should mention it here given it will improve performance esp. on bigger projects.

I tried it out on bigger projects and I ran into limits with the default size. Therefore I made it more configurable but the improvement hasn't been released yet.